app/src/main/java/com/example/marcinwisniewski/intivetrainingapp/movielist/viewmodel/ListViewModel.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.marcinwisniewski.intivetrainingapp.movielist.viewmodel
+
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.Movie
+import com.example.marcinwisniewski.intivetrainingapp.movielist.model.MovieProvider
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+
+class ListViewModel : ViewModel() {
+    private var moviesList: MutableLiveData<List<Movie>>? = null
+    fun getMovies(): MutableLiveData<List<Movie>>? {
+        if (moviesList == null) {
+            moviesList = MutableLiveData()
+            MovieProvider.movieProvider.getMoviesResponse()?.subscribeOn(Schedulers.io())
```

Comment:
Please format this Rx sequence. It would be look better when each operator is in new line

Reply:
fixed